### PR TITLE
20 logic graph overwrite feature eg fpr gpio handling example

### PR DESCRIPTION
**Pull request by Galoshi**

### CODE REVIEW

File: source/pcic/client.py [hunk 1]
```diff
@@ -26,11 +26,11 @@ def __init__(self, address, port, autoconnect=True, timeout=5):
...
-    def __enter__(self):
-        return self
-
-    def __exit__(self, exc_type, exc_val, exc_tb):
-        self.disconnect()
+    # def __enter__(self):
+    #     return self
+    #
+    # def __exit__(self, exc_type, exc_val, exc_tb):
+    #     self.disconnect()
```
Comment (cfreundl) [inline source/pcic/client.py:33]: Is this change intentional? If yes, then better remove the lines instead of commenting them out.

Comment (Galoshi) [reply source/pcic/client.py]: Fixed in commit 231f3fc9e5c9a4bf55caddcc65ac70e49a27ace4

Comment (cfreundl) [reply source/pcic/client.py]: Looks good to me, PR is ready for merge I would say.
